server/views/partials/layout.njk

```diff
@@ -58,4 +58,8 @@
   <script src="/assets/govukFrontendInit.js"></script>
   <script src="/assets/moj/all.js"></script>
   <script src="/assets/mojFrontendInit.js"></script>
+  {% block pageScripts %}
+     <script src="/assets/card.js"></script>
```

Comment:
I think this can just be an empty block i.e. 
{% block pageScripts %}
{%  endblock %}

as we load the card.js file into the block in the homepage.njk don't we? We don't necessarily want the card.js file being loaded on every page which is what this is doing